### PR TITLE
Add manual testing docs for DGS Reduction

### LINKED ISSUE #37387: Update Manual Testing DGSReduction Instructions
**Describe the issue.**

The instructions for manual testing of DGSReduction interface need to be updated and the links in the instructions have to redirect to existing web pages.

Observed during manual testing in #37343

### PR DESCRIPTION
**Pull request by koshchii**
### Description of work

This PR adds manual testing instructions for the DGS Reduction interface.

#### Summary of work

Fixes #37387

### To test:

1. Build the dev-docs-html target
2. Check the docs are formatted correctly when opened in a browser and the instructions make sense.
3. The sample data for testing the instructions: [SampleData-MLZ.zip](https://github.com/user-attachments/files/15616280/SampleData-MLZ.zip)

*This does not require release notes* because **it is part of the developer documentation only.**

---

### Reviewer

Please comment on the points listed below ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)).
**Your comments will be used as part of the gatekeeper process, so please comment clearly on what you have checked during your review.** If changes are made to the PR during the review process then your final comment will be the most important for gatekeepers. In this comment you should make it clear why any earlier review is still valid, or confirm that all requested changes have been addressed.

#### Code Review

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Do the release notes conform to the [release notes guide](https://developer.mantidproject.org/Standards/ReleaseNotesGuide.html)?

#### Functional Tests

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### Gatekeeper

If you need to request changes to a PR then please add a comment and set the review status to "Request changes". This will stop the PR from showing up in the list for other gatekeepers.

### CODE REVIEW

File: dev-docs/source/Testing/Direct/DGSReductionTests.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+---------------
+
+- Check ``Subtract empty can from vanadium``.
+- Normalise ``to monitor``.
+- Correct TOF ``vanadium``.
+
+Data section
+------------
+
+- First row: Type ``27:29`` in ``Data runs`` column and ``H2O_21C`` in ``Comment``.
```
Comment (adriazalvarez) [inline dev-docs/source/Testing/Direct/DGSReductionTests.rst:45]: Can you add another entry specifying that the second row does appear until you finish typing data on the first `Data Runs` cell?

File: dev-docs/source/Testing/Direct/DGSReductionTests.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+- Set Q ``start`` value to ``0.400``, ``step`` to ``0.1``, and end to ``2``.
+
+Options section
+---------------
+
+- Check ``Subtract empty can from vanadium``.
+- Normalise ``to monitor``.
+- Correct TOF ``vanadium``.
+
+Data section
```
Comment (adriazalvarez) [inline dev-docs/source/Testing/Direct/DGSReductionTests.rst:42]: Can you rename this section to Data Table Section?

File: dev-docs/source/Testing/Direct/DGSReductionTests.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+------------
+
+- First row: Type ``27:29`` in ``Data runs`` column and ``H2O_21C`` in ``Comment``.
+- Second row: Type ``30:31`` in ``Data runs`` column and ``H2O_34C`` in ``Comment``.
+
+Data reduction
+--------------
+
+- Click ``Reduce`` button to start the data reduction.
+- The workspaces with reduced data will appear in the ``Workspaces`` panel of Workbench. If default workspace prefix ``ws`` was used, the final result should be saved to the workspace group called ``gwsDataSQW``.
```
Comment (adriazalvarez) [inline dev-docs/source/Testing/Direct/DGSReductionTests.rst:52]: Could you add a small sentence saying how many workspaces are expected based on the input, and wether they are groups or not?

**Review by adriazalvarez — CHANGES_REQUESTED** [review]
Thanks for redacting these instructions. 
I've built the dev-docs with no issue. Formatting is ok. 
Followed test instructions, and found no inconsistencies between the instructions and the results. 

Even though instructions are ok and easy to follow, I have added a few suggestions for changes that I think will speed up the process of manual testing for developers. 

Other than that, would you mind adding another section, even if very succint, with instructions for saving?

**Comment by koshchii** [comment]
Thanks for the review and suggestions! The new commit contains the requested changes

**Review by adriazalvarez — APPROVED** [review]
Thanks for the changes, looks good to me.

**Pull request merged**